tests/continental_extension_mixed_compositions_particles.prm

```diff
@@ -0,0 +1,378 @@
+# This is a simplified version of the continental extension cookbook to test
+# the combination of different compositional field discretizations and
+# particles
+
+####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 10000
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 5
+set CFL number                             = 0.25
+set Maximum time step                      = 10e3
+set Pressure normalization                 = no
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block AMG
+    set Number of cheap Stokes solver steps = 4000
+    set Linear solver tolerance             = 1e-8
+    set GMRES solver restart length         = 100
+    set Use full A block as preconditioner  = true
+  end
+
+  subsection Newton solver parameters
+    set Max pre-Newton nonlinear iterations      = 10
+    set SPD safety factor                        = 0.9
+    set Nonlinear Newton solver switch tolerance = 1e-5
+    set Max Newton line search iterations        = 5
+    set Maximum linear Stokes solver tolerance   = 1e-8
+    set Use Newton residual scaling method       = true
+    set Use Newton failsafe                      = true
+    set Stabilization preconditioner             = SPD
+    set Stabilization velocity block             = SPD
+    set Use Eisenstat Walker method for Picard iterations = false
+  end
+end
+
+# Governing equations
+subsection Formulation
+  set Formulation = Boussinesq approximation
+  set Enable elasticity    = false
+end
+
+subsection Discretization
+  set Composition polynomial degree           = 1,1,1,2,1
```

Comment:
```suggestion
 # As specified in the Compositional field subsection, the first three
 # fields are tracked by particles and the last two are tracked by fields.
 # The Compositional polynomial degree must be the same for all
 # compositions tracked by particles, while it can vary for compositions
 # tracked by fields. 
  set Composition polynomial degree           = 1,1,1,2,1
```